scripts/ci/update_server_names.py

```diff
@@ -61,8 +61,15 @@ def fetch_server_list():
     string_map = countries
     for city in cities:
         # Remove state suffix, capitalize each work, remove spaces.
-        id = city.split(",")[0].strip().title().replace(" ", "")
-        string_map[id] = city
+        baseId = city.split(",")[0].strip().title().replace(" ", "")
```

Comment:
You should be able to obtain the same with this code

```
        id = city.split(",")[0].strip().title().replace(" ", "")
        id = re.sub(r'[^A-Za-z\s]', '', id)
        string_map[id] = city
```

Reply:
Good call, thanks! Updated this. 

I didn't include the white space check (`\s`), as I don't think we need it. We remove all spaces in the first line (`.replace(" ", "")`). While no current city names have any other whitespace characters, we'd want to remove them if they did.